src/LinAlg/matrix.h

```diff
@@ -387,7 +387,7 @@ namespace utl //! General utility classes and functions.
         for (size_t r = 0; r < ncol; r++)
           for (size_t c = 0; c < nrow; c++)
             this->elem[c+nrow*r] = mat.elem[r+ncol*c];
-      else
+      else if (!mat.elem.empty())
         this->elem.fill(mat.elem.ptr());
```

Comment:
and maybe use `data()` here as well, for consistency.

Reply:
ok